code/game/gamemodes/vampire/vampire.dm

```diff
@@ -419,7 +426,7 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 	if(istype(owner.loc, /turf/space))
 		check_sun()
 	if(istype(owner.loc.loc, /area/chapel) && !get_ability(/datum/vampire_passive/full))
-		vamp_burn(0)
+		vamp_burn(7)
```

Comment:
This also makes the vamps burn in the chapel. Be sure to include this in your PR if it's intended

Reply:
The behaviour in the chapel has not changed. I just adjusted how the vamp_burn proc's argument works. Previously, you used vamp_burn(0) for a weak burn which had a 7% chance of doing something and vamp_burn(1) for a strong burn with 34% chance of doing something. Since that was very stupid, I made it so that vamp_burn(7) now has a 7% chance of doing something, etc.

In other words, chapel behaviour is exactly the same.